src/Compilers/VisualBasic/Portable/Symbols/NamedTypeSymbol.vb

```diff
@@ -1272,8 +1272,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
             ' Should this be optimized for perf (caching for VT<0> to VT<7>, etc.)?
             If Not IsUnboundGenericType AndAlso
-                ContainingSymbol?.Kind = SymbolKind.Namespace AndAlso
-                ContainingNamespace?.ContainingNamespace?.IsGlobalNamespace = True AndAlso
+                (ContainingSymbol?.Kind = SymbolKind.Namespace).GetValueOrDefault() AndAlso
+                (ContainingNamespace.ContainingNamespace?.IsGlobalNamespace).GetValueOrDefault() AndAlso
```

Comment:
>(ContainingNamespace.ContainingNamespace?.IsGlobalNamespace).GetValueOrDefault() [](start = 16, length = 80)

This is a fix, we are going to dereference ContainingNamespace below as the last condition even when it was null. Prior to this PR, compiler would incorrectly apply GetValueOrDefault call itself.